doctr/datasets/datasets/base.py

```diff
@@ -49,6 +53,18 @@ def __getitem__(
             # typing issue cf. https://github.com/python/mypy/issues/5485
             img = self.sample_transforms(img)  # type: ignore[call-arg]
 
+        if isinstance(target, dict) and 'boxes' in target.keys() and self.convert_to_relative:
```

Comment:
No need to check for something else than `self.convert_to_relative` considering we're unifying the target format